frontend/src/views/detail/hook/useDetailBook.ts

```diff
@@ -70,13 +73,23 @@ export const useDetailBook = (): TUseDetailBook => {
 
       // Delete the book
       await API.deleteBook(id);
+      addToast({
```

Comment:
Toast behaviour is not a hook responsibility, please move this code in a different file